### ISSUE TITLE
Add IC ID in IC selection of component config editor

### ISSUE THREAD

**Issue by iripiri** [VILLASframework/web#437]
When editing a component config, it can be hard to know which IC to select if there are several ICs with the same name (see screenshot).
I propose to show the selectable ICs as "IC name [IC ID: ..]"

![grafik](https://github.com/user-attachments/assets/ef6a7b5e-2a6b-4b53-8c25-b9187967c182)

**Comment by stv0g** [comment]
Yeah, I had the same issue.. But I dont know if we really want to burden the user with UUIDs? Maybe a more general problem is that those ICs have the name? But I dont have a good idea how we could avoid this.

What do you think about, just showing the UUID in the drop-down, in case the names are non-unique?

**Comment by iripiri** [comment]
I was thinking of the simple database IC IDs that you can also see in the Infrastructure Components table. I guess IC IDs as well as UUIDs are both only known to administrators? But the IC IDs at least tell you which one is older or newer

**Comment by SystemsPurge** [comment]
Maybe it would be helpful to have the creation date ( + hour:min ) aligned to the right. It helps keep track over a short period of time at least
